### ISSUE TITLE
sync GitHub repo with FH2 SVN repo

### ISSUE THREAD

**Issue by ByteHazard** [ByteHazard/BfMeshView#2]
Please note: the code here on GitHub is out of date, and I need to synchronize it with the code currently kept in the FH2 development SVN repo.

**Comment by rPoXoTauJIo** [comment]
Open for contributions after sync, or fh2 svn repo adding features faster? :)

**Comment by ByteHazard** [comment]
Yes, I would like to do that. But I have to figure out a way to sync this git/svn repo automatically (and without problems) first.

**Comment by rPoXoTauJIo** [comment]
Same way like PR localization done? Externals: https://i.imgur.com/pbQzWE3.png
https://github.com/realitymod/pr-localization

**Comment by SpencerSharkey** [comment]
Excited to see what's been changed since the initial release 👍
